### PR TITLE
Update to x265 multilib

### PR DESCRIPTION
**Pull request by ggoussard**
Add support of X265 multilib (ie. 8bits, 10bits and 12bits).

```
docker run -it -v $PWD:/tmp/workdir ade6e7dca1be -stats -t 00:00:10  -i /tmp/workdir/tearsofsteel_4k.mov -c:v:0 libx265 -pix_fmt yuv420p12 -preset slow -an -dn -sn -f mp4 test_12bits.mp4

ffprobe -hide_banner test_12bits.mp4
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'test_12bits.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2mp41
    encoder         : Lavf57.41.100
  Duration: 00:00:10.00, start: 0.000000, bitrate: 2518 kb/s
    Stream #0:0(eng): Video: hevc (Rext) (hev1 / 0x31766568), yuv420p12le(tv, bt2020nc/bt2020/smpte2084), 3840x1714 [SAR 1:1 DAR 1920:857], 2514 kb/s, 24 fps, 24 tbr, 12288 tbn, 24 tbc (default)
    Metadata:
      handler_name    : VideoHandler


```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -213,23 +202,7 @@ RUN     export MAKEFLAGS="-j$[$(nproc) + 1]" && \
...
-        libtool \
-        libssl-dev \
-        make \
-        nasm \
-        perl \
-        pkg-config \
-        python \
-        xmlto \
-        zlib1g-dev && \
+        apt-get purge -yqq $buildDeps\
```
Comment (jrottenberg) [inline Dockerfile]: missing the && \

File: Dockerfile [hunk 2]
```diff
@@ -25,160 +25,149 @@ ENV         FFMPEG_VERSION=3.1.2 \
             VORBIS_VERSION=1.3.5 \
             VPX_VERSION=1.6.0    \
             XVID_VERSION=1.3.4   \
-            X265_VERSION=1.9     \
+            X265_VERSION=2.0     \
+            X264_VERSION=last_stable \
```
Comment (jrottenberg) [inline Dockerfile]: I'd put a pinned version here , like 20160827 to get idempotent build

**Comment by jrottenberg** [comment]
ah failed on 

`      &&         apt-get purge -yqq $buildDeps        apt-get autoremove -y &&      
`

Put the comment inline, thannks a lot, it looks fantastic, solving bunch of pending tasks on my todo list

**Comment by jrottenberg** [comment]
Fantastic ! Thanks a lot

**Comment by ggoussard** [comment]
pix-fmt: yuv420p8 yuv420p10 yuv420p12 for 8bits, 10bits or 12bits output.

**Pull request merged**